config/rbac/role.yaml

```diff
@@ -17,6 +17,26 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - batch
+  resources:
+  - jobs
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
```

Comment:
is this section needed? I don't think there's a batch/pod resource in API, is it?

Reply:
Done